templates/iptables.rules.tmpl

```diff
@@ -3,14 +3,14 @@
 :INPUT ACCEPT [0:0]
 :OUTPUT ACCEPT [0:0]
 :POSTROUTING ACCEPT [0:0]
--A POSTROUTING -s {{.DockerSubnet}} ! -d {{.DockerSubnet}} -j MASQUERADE
+{{if .UseDockerRules}}-A POSTROUTING -s {{.DockerSubnet}} ! -d {{.DockerSubnet}} -j MASQUERADE{{end}}
 COMMIT
 
 *filter
 :INPUT DROP [0:0]
 :FORWARD DROP [0:0]
 :OUTPUT DROP [0:0]
-:DOCKER - [0:0]
+{{if .UseDockerRules}}:DOCKER - [0:0]
```

Comment:
Shouldn't the if be closed here?

Like this: `{{if .UseDockerRules}}:DOCKER - [0:0]{{end}}`